csv-generator/runtime/src/main/java/io/quarkiverse/operatorsdk/csv/runtime/CSVMetadata.java

```diff
@@ -53,10 +53,12 @@
 
         String[] resources();
 
-        // By default, all verbs
-        String[] verbs() default {};
+        String[] verbs() default { "get", "list", "watch", "create", "delete", "patch", "update" };
```

Comment:
Ah, I forgot that we can use a constant array for this… 😢

Reply:
I finally had to provide the default list of verbs also here: https://github.com/quarkiverse/quarkus-operator-sdk/pull/238/files#diff-8822dba1a9957d5d2fc78cd41128146a273ebc1bef596c3de9675d8479c35834R249
Otherwise, when getting the default values from the annotations, it returns no verbs and hence the build fails.